crates/egui/src/widgets/plot/items/mod.rs

```diff
@@ -139,6 +143,12 @@ impl HLine {
         self
     }
 
+    /// Hidden on initialization. Default is true.
+    pub fn visible(mut self, visible: bool) -> Self {
```

Comment:
```suggestion
    /// Is the item visible on initialization? Default is `true`.
    #[inline]
    pub fn visible(mut self, visible: bool) -> Self {
```

Reply:
and everywhere else

Reply:
Also, I think this is better named `default_visible`, since it shouldn't control the current visibility, just the visibility on the first time we show the plot

Reply:
Thank ! I did not suspect that it was necessary to add [#inline]  and renamed to default_visible which indicates well that it is an initialization parameter.